draft-irtf-cfrg-opaque.md

```diff
@@ -876,53 +1012,52 @@ messages and the KE session.
 We use the parameters Npk and Nsk to denote the size of the public and private keys used
 in the AKE instantiation.
 
-The rest of this section includes key schedule utility functions used by OPAQUE-3DH,
-and then provides a detailed specification for OPAQUE-3DH, including its wire format
-messages.
+This specification defines OPAQUE with the 3DH AKE. Alternative AKEs are presented in {{alternate-akes}}.
 
-### Key Schedule Utility Functions
+### OPAQUE-3DH Instantiation {#opaque-3dh}
 
-The key derivation procedures for OPAQUE-3DH makes use of the functions below, re-purposed
-from TLS 1.3 {{?RFC8446}}.
+OPAQUE-3DH is implemented using a suitable prime order group. All operations in
+the key derivation steps in {{derive-3dh}} are performed in this group and
+represented here using multiplicative notation. The output of OPAQUE-3DH is a
+session secret `session_key` and export key `export_key`.
 
-~~~
-Expand-Label(Secret, Label, Context, Length) =
-  Expand(Secret, CustomLabel, Length)
-~~~
+OPAQUE-3DH allows the server to send additional encrypted and authenticated application specific information.
+An opaque byte string `server_info` is encrypted using a one-time-pad derived from Ke2 and then used as input to XOR
+with the `server_info` plaintext.
 
-Where CustomLabel is specified as:
+The parameters Npk and Nsk are set to be equal to the size of an element and
```

Comment:
Should we reference the dependencies section, where this is specified?

Reply:
Maybe this phrase is not even necessary, we could drop it